modules/installation-bootstrap-gather.adoc

```diff
@@ -64,6 +66,7 @@ A default cluster contains three control plane machines. List all of your contro
 +
 The command output resembles the following example:
```

Comment:
You could change this to `.Example output`

Reply:
Resolved.